doc/source/user_guide/merging.rst

```diff
@@ -88,7 +88,9 @@ some configurable handling of "what to do with the other axes":
   concatenation axis does not have meaningful indexing information. Note
   the index values on the other axes are still respected in the join.
 * ``join_axes`` : list of Index objects. Specific indexes to use for the other
```

Comment:
you can completely remove the join_axes here; its deprecated and does not need to appear here at all; its in the doc-string which is enough

Reply:
can you update this